tests/baselines/reference/classExpressionWithStaticPropertiesES64.js

```diff
@@ -0,0 +1,10 @@
+//// [classExpressionWithStaticPropertiesES64.ts]
+(class { static x = 0; });
+
+
+//// [classExpressionWithStaticPropertiesES64.js]
+((classExpression_1 = class {
+    },
+    classExpression_1.x = 0,
+    classExpression_1));
+var classExpression_1;

```

Comment:
Why is it polluting the global scope with classExpression_1?

Can this be hidden in a self-executing function?